### PR TITLE
CIV-7978 SDO Toggle Cleanup 

### PR DESCRIPTION
**Pull request by LTurkingtonHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-7978

### Change description ###
Removed SDO toggle journeys where toggle set to false in claimant response and claimant response spec
Updated tests 

Claimant response unspec master 
image

Claimant response unspec with SDO removed 
image

Claimant response spec master 
image

Claimant response spec with SDO removed 
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by HarryH96 — APPROVED** [review]

**Review by sankaviv1 — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseTest.java [hunk 1]
```diff
@@ -29,97 +29,13 @@ class ClaimantResponseTest extends BpmnBaseTest {
...
-    private static final String APPLICATION_PROCEEDS_IN_HERITAGE_ACTIVITY_ID_1 = "UpdateGeneralApplicationStatus1";
     public static final String APPLICATION_OFFLINE_UPDATE_CLAIM = "APPLICATION_OFFLINE_UPDATE_CLAIM";
     private static final String APPLICATION_OFFLINE_UPDATE_CLAIM_ACTIVITY_ID = "UpdateClaimWithApplicationStatus";
-    private static final String APPLICATION_OFFLINE_UPDATE_CLAIM_ACTIVITY_ID_1 = "UpdateClaimWithApplicationStatus1";
 
     public ClaimantResponseTest() {
         super("claimant_response.bpmn", "CLAIMANT_RESPONSE_PROCESS_ID");
     }
 
-    @Test
```
Comment (sherlynkhaw) [inline src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseTest.java:41]: why is this removed? is it a redundant test? the one below as well.

**Review by HarryH96 — APPROVED** [review]
